pubsub: test constants fail on 32 bit systems

```
# cloud.google.com/go/pubsub [cloud.google.com/go/pubsub.test]
src/cloud.google.com/go/pubsub/flow_controller_test.go:230:28: cannot use 4e9 (untyped float constant 4e+09) as int value in argument to fc.acquire (truncated)
src/cloud.google.com/go/pubsub/flow_controller_test.go:235:28: cannot use 4e9 (untyped float constant 4e+09) as int value in argument to fc.acquire (truncated)
# cloud.google.com/go/pubsub/internal/longtest_test [cloud.google.com/go/pubsub/internal/longtest.test]
src/cloud.google.com/go/pubsub/internal/longtest/endtoend_test.go:326:39: constant 30000000000 overflows int
```

--- a/pubsub/flow_controller_test.go
+++ b/pubsub/flow_controller_test.go
@@ -226,13 +226,13 @@ func TestFlowControllerUnboundedBytes(t *testing.T) {
 	ctx := context.Background()
 	fc := newFlowController(fcSettings(2, 0, FlowControlSignalError))
 
-	// Successfully acquire 4GB.
-	if err := fc.acquire(ctx, 4e9); err != nil {
+	// Successfully acquire 2GB.
+	if err := fc.acquire(ctx, 2e9); err != nil {
 		t.Errorf("got %v, wanted no error", err)
 	}
 
-	// Successfully acquired 4GB bytes.
-	if err := fc.acquire(ctx, 4e9); err != nil {
+	// Successfully acquired 2GB bytes.
+	if err := fc.acquire(ctx, 2e9); err != nil {
 		t.Errorf("got %v, wanted no error", err)
 	}
 

--- a/pubsub/v2/flow_controller_test.go
+++ b/pubsub/v2/flow_controller_test.go
@@ -226,13 +226,13 @@ func TestFlowControllerUnboundedBytes(t *testing.T) {
 	ctx := context.Background()
 	fc := newFlowController(fcSettings(2, 0, FlowControlSignalError))
 
-	// Successfully acquire 4GB.
-	if err := fc.acquire(ctx, 4e9); err != nil {
+	// Successfully acquire 2GB.
+	if err := fc.acquire(ctx, 2e9); err != nil {
 		t.Errorf("got %v, wanted no error", err)
 	}
 
-	// Successfully acquired 4GB bytes.
-	if err := fc.acquire(ctx, 4e9); err != nil {
+	// Successfully acquired 2GB bytes.
+	if err := fc.acquire(ctx, 2e9); err != nil {
 		t.Errorf("got %v, wanted no error", err)
 	}
 

